### ISSUE TITLE
[Feature Request] New Poweritems

### ISSUE THREAD

**Issue by ChikoEduardo** [Regalis11/Barotrauma#1923]
I had these in my mod. They no longer work like they used to so I'll suggest them since they were useful ítems, IMO. :3

Circuit Breaker:
A PowerTransfer component with little Health and a lowered OverloadVoltage so when it takes damage, it "shuts itself down" first, preventing the rest of the grid from taking severe damage on power spikes. A toggle function is really useful in them too, to allow ship builders to wire buttons to it so crew can turn them on and off at will.

Power Distribution Units:
Basically, smaller Junction Boxes with extra power outlets but no signal ones. I would attach these next to junction boxes for the extra outlets. Having the power_in and power_out outlets would make them more versatile tan just the power one.

It would also be cool if we could edit the number of wires allowed in power outlets or add more power outlets.

Here's a pic with the ones I used in the mod as a way to illustrate this request:
https://i.imgur.com/uvFiKR3.jpg

**Comment by Regalis11** [comment]
Issue moved to [Regalis11/Barotrauma-development #1715](https://github.com/Regalis11/Barotrauma-development/issues/1715) via [**ZenHub**](https://www.zenhub.com/)
